src/DocumentDB.ChangeFeedProcessor/Exceptions/LeaseLostException.cs

```diff
@@ -8,46 +8,86 @@ namespace Microsoft.Azure.Documents.ChangeFeedProcessor.Exceptions
     using System.Runtime.Serialization;
     using Microsoft.Azure.Documents.ChangeFeedProcessor.PartitionManagement;
 
+    /// <summary>
+    /// Exception occurred when lease was taken by a different host.
```

Comment:
Please change this to more generic, like: ... when lease is lost, that would typically happen when it is taken by another host. Other cases: at least: communication failure, number of retries reached.

Reply:
Sure, updated.